app/frontend/src/features/search/SearchResult.tsx

```diff
@@ -16,77 +18,103 @@ import {
   LabelsReferencesLastActive,
 } from "features/user/UserTextAndLabel";
 import { User } from "pb/api_pb";
-import { Link } from "react-router-dom";
-import { routeToUser } from "routes";
+import LinesEllipsis from "react-lines-ellipsis";
 import makeStyles from "utils/makeStyles";
 
 const useStyles = makeStyles((theme) => ({
-  card: {
-    borderRadius: theme.shape.borderRadius * 3,
+  root: {
+    textDecoration: "none",
+    // width: "100%",
   },
   about: {
     margin: `${theme.spacing(2)} 0`,
   },
+  card: {},
   statusLabelWrapper: {
     display: "flex",
     "& > div": {
       display: "flex",
     },
   },
-  statusLabel: {
-    marginLeft: theme.spacing(1),
-    marginRight: theme.spacing(2),
+  statusIcon: {
+    marginInlineEnd: theme.spacing(1),
   },
-  root: {
-    [theme.breakpoints.up("sm")]: {
-      width: "49%",
-    },
-    marginBottom: theme.spacing(2),
-    textDecoration: "none",
-    width: "100%",
+  statusLabel: {
+    marginInlineEnd: theme.spacing(2),
   },
 }));
 
-export default function SearchResult({ user }: { user: User.AsObject }) {
+interface SearchResultProps {
+  className?: string;
+  id?: string;
+  user: User.AsObject;
+  onClick: (user: User.AsObject) => void;
+  highlight?: boolean;
+}
+
+export default function SearchResult({
+  className,
+  id,
+  user,
+  onClick,
+  highlight = false,
+}: SearchResultProps) {
   const classes = useStyles();
   return (
-    <Link to={routeToUser(user.username)} className={classes.root}>
-      <Card className={classes.card}>
-        <CardActionArea>
-          <CardContent>
-            <UserSummary user={user}>
-              <div className={classes.statusLabelWrapper}>
-                <div>
-                  <CouchIcon />
-                  <Typography
-                    className={classes.statusLabel}
-                    display="inline"
-                    variant="subtitle1"
-                    color="primary"
-                  >
-                    {hostingStatusLabels[user.hostingStatus]}
-                  </Typography>
-                </div>
-                <div>
-                  <LocationIcon />
-                  <Typography
-                    className={classes.statusLabel}
-                    display="inline"
-                    variant="subtitle1"
-                  >
-                    {meetupStatusLabels[user.meetupStatus]}
-                  </Typography>
-                </div>
+    <Card
+      id={id}
+      className={classNames(classes.card, classes.root, className)}
```

Comment:
```suggestion
      className={classNames(classes.root, className)}
```
Unneeded `classes.card` since no CSS are in the style object there?